src/k8s/cmd/k8s/k8s_bootstrap_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"testing"
 
 	apiv1 "github.com/canonical/k8s-snap-api/api/v1"
+	apiv1_annotations "github.com/canonical/k8s-snap-api/api/v1/annotations"
```

Comment:
nit: Maybe `annotationsv1`?

Reply:
I don't think this is a good idea. The `corev1` and `metav1` conventions are there because we have `/core/v1` and `/meta/v1`. I guess `apiv1_annotations` is a better choice. I'm OK with both TBH but that's completely up to you. Sorry for the misleading comment.